misc/10-aws.conflist

```diff
@@ -13,7 +13,7 @@
     {
       "name": "egress-v4-cni",
       "type": "egress-v4-cni",
-      "mtu": 9001,
+      "mtu": "9001",
```

Comment:
why this change?

Reply:
This was messing up the json conversion.